ruby.rb

```diff
@@ -0,0 +1,21 @@
+class Yo
+  def hello(name)
+    "hello there #{name}!"
+  end
+
+  def faillo(name)
+    'hello there! ' + name.to_s
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.